app.ts

```diff
@@ -0,0 +1,473 @@
+const headings: any = document.querySelector("#Headings");
+const content_cols: any = document.querySelector("#Content");
+const pageStats: any = document.getElementById('pageStats')
+const nextButton: any = document.querySelector("#next");
+const prevButton: any = document.querySelector("#prev");
+const clear = "";
+let paramOne: any = "0"
+let paramTwo: any = "9"
```

Comment:
I think rather calculate this value based on the size of the window than have a fixed value. Screen sizes change and you can potentially fit more records on the screen. Try to fit as much readable records on the screen as possible.